config/deployment/dev/dispatch.yaml

```diff
@@ -14,6 +14,8 @@ dispatch:
     service: osteo
   - url: "testboston.dev.datadonationplatform.org/*"
     service: testboston
+  - url: "mpc.dev.datadonationplatform.org/*"
```

Comment:
@zyme, I've added a new URL in the `dispatch.yaml`, do we need to add this URL somewhere else?

Reply:
Could you add it to all environments while you're at it?  Production is `mpcproject.org`.  Since this  routing  won't be used by prod  until we  update  DNS, it's safe  to  add  it  now.